cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
Done in https://github.com/cmus/cmus/commit/7b5297a5b9b0690901b2c129056b877ed40dc160.